### ISSUE TITLE
Version 1.0 release?

### ISSUE THREAD

**Issue by freundTech** [75th/mario-maker-linker#17]
Now that Issue #11 has been fixed do you think we should bump the Version to v1.0 and call the addon feature complete (for now) or is there anything else that should be done before?

**Comment by freundTech** [comment]
I just merged the changes which have been in dev for almost 2 years into master, did a bit of cleanup and bug fixing and published it as version 1.0 on Mozilla Addons.

All the Firefox specific code was no longer needed as newer versions of Firefox have Webextension support (And don't support the old Firefox addon format anymore), so both the Chrome and Firefox version now share 100% of the codebase.

I know this has been inactive for a really long time, but maybe you could also push the update to the Chrome addon site? @75th

**Comment by 75th** [comment]
I would love to, and I'm sorry for abandoning this for so long; I do have a habit of doing that. I'll try to  get it packaged and published this weekend.
